auth/src/main/java/com/firebase/ui/auth/ui/AppCompatBase.java

```diff
@@ -53,4 +59,9 @@ protected void switchFragment(@NonNull Fragment fragment,
     protected void switchFragment(@NonNull Fragment fragment, int fragmentId, @NonNull String tag) {
         switchFragment(fragment, fragmentId, tag, false, false);
     }
+
+    @SuppressLint("all")
```

Comment:
Do we actually need this `@SuppressLint` anymore?  If so can we be more specific than `all`?

Reply:
So then could it be `@SuppressLint("SourceLockedOrientationActivity")`?